packages/requisitions/src/SupplierRequisition/ListView/Toolbar.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, useEffect, useRef } from 'react';
+import {
+  useNotification,
+  DropdownMenu,
+  DropdownMenuItem,
+  useTranslation,
+  DeleteIcon,
+  useTableStore,
+  AppBarContentPortal,
+  SearchBar,
+  FilterController,
+} from '@openmsupply-client/common';
+import { RequisitionRow } from '../../types';
+
+export const Toolbar: FC<{
+  onDelete: (toDelete: RequisitionRow[]) => void;
+  filter: FilterController;
+  data?: RequisitionRow[];
+}> = ({ onDelete, data, filter }) => {
+  const t = useTranslation('distribution');
+
+  const { success, info } = useNotification();
+
+  const { selectedRows } = useTableStore(state => ({
+    selectedRows: Object.keys(state.rowState)
+      .filter(id => state.rowState[id]?.isSelected)
+      .map(selectedId => data?.find(({ id }) => selectedId === id))
+      .filter(Boolean) as RequisitionRow[],
+  }));
+
+  const deleteAction = () => {
+    if (selectedRows && selectedRows?.length > 0) {
+      onDelete(selectedRows);
+      success(`Deleted ${selectedRows?.length} invoices`)();
```

Comment:
:nit translations for both of these

Reply:
Ah yes. I think i'll make an issue (I need to do the invoices also) - I realised I was also a little unsure of what to do for the new translation file - since we have `distribution`, do you think`replenishment` and then add `deleted X invoices` to both distribution and replenishment? 🤔

Reply:
yeah, that sounds good. I was in between two approaches: having anything that is used in more than one area put in a shared file - and keeping as much as possible in each separate area. For this one - the line is pretty specific, so I would duplicate it, and have in both distribution and replenishment. That's where I'm leaning now - all the really general stuff in common, and specific things in the file for each area, with duplication as necessary 🤷

Reply:
Ah great that sounds good. One other thought I had was that the columns themselves use a locale key without the namespace specified. So we could either somehow put into the column definition a namespace or use a static namespace for all columns.. something just like a `datatable` namespace which the `column` component uses to translate keys for example

Reply:
I do like having the locale key on the column, rather than a string. Could have a `datatable` namespace, though I like the idea of having all the translations for a functional area together in the same file. Could pass in the namespace to the column somehow though then the component is tied to a specific implementation of localisation. Could do something hokey with the locale key, say `[namespace]_[localy-key]` but that is bad on even more levels.

I would go with (a) a `datatable` namespace or (b) keep them all in `common` namespace - but really don't mind whatever way. Can't see an ideal option.